common/openapi.yml

```diff
@@ -90,6 +90,29 @@ paths:
         '500':
           $ref: '#/components/responses/ServerError'
 
+  /api/oidc/v1/{provider}/accessToken:
```

Comment:
ack, it actually looks like ecm kebab-cases its URLs! Lets do that to be consistent... Sorry!

Reply:
I think we should also update oauthcode to be oauth-code (the UI is the only thing calling that right?)

Reply:
I think we can just leave that one, since we want to get this out in front of Workflows ASAP. We can come back to it.

Reply:
yeah agreed best not to do it now, let's just track that as one of our follow-up items